src/components/Home.tsx

```diff
@@ -1,14 +1,18 @@
-import React from 'react';
+import React, { useState } from 'react';
 
 import Carousel from './Carousel';
 import Base from './shared/Base';
 import { HeaderSections } from './shared/PlaynetConstants';
+import YouTube from './Youtube';
 
 function Home(): JSX.Element {
+  const [ chosenVideoPath, chooseVideo ] = useState(null);
+  const [ showCarousel, toggleCarousel ] = useState(false);
+
   return (
     <div>
       <Base section={HeaderSections.INTRO}>
-        <Carousel
+        { showCarousel ? <Carousel
```

Comment:
nit, not important, feel free to dismiss: can we make the carousel on a new line so that its easier to read?
```suggestion
        { showCarousel ? 
           <Carousel
```